app/screens/channel_info/channel_info.js

```diff
@@ -312,19 +333,31 @@ class ChannelInfo extends PureComponent {
                             <View>
                                 <View style={style.separator}/>
                                 <ChannelInfoRow
-                                    action={() => preventDoubleTap(this.goToChannelAddMembers)}
+                                    action={this.goToChannelAddMembers}
                                     defaultMessage='Add Members'
                                     icon='user-plus'
                                     textId='channel_header.addMembers'
                                     theme={theme}
                                 />
                             </View>
                         }
+                        {canEditChannel && (
+                            <View>
+                                <View style={style.separator}/>
+                                <ChannelInfoRow
+                                    action={this.handleChannelEdit}
+                                    defaultMessage='Edit Channel'
+                                    icon='edit'
+                                    textId='channel_header.mobile.edit'
+                                    theme={theme}
+                                />
+                            </View>
+                        )}
                         {this.renderLeaveOrDeleteChannelRow() &&
                             <View>
                                 <View style={style.separator}/>
                                 <ChannelInfoRow
-                                    action={() => preventDoubleTap(this.handleDeleteOrLeave, this, 'leave')}
+                                    action={() => this.handleDeleteOrLeave('leave')}
```

Comment:
is there a way we can avoid the arrow function here?

Reply:
The only way I can think of is to add a `this.handleDelete` and `this.handleLeave` that wraps the call to `this. handleDeleteOrLeave `.  Adds extra boilerplate code but if we want to avoid the arrow function here that would work

Reply:
yeah perhaps two different functions, one for leave and one for delete that in turn call this.handleDeleteOrLeave